packages/operation/src/evm/deploy/providers.ts

```diff
@@ -0,0 +1,45 @@
+import { ethers } from 'ethers';
+import { deriveProviderId } from '../utils';
+import { DeployState as State } from '../../types';
+
+export async function createProviders(state: State): Promise<State> {
+  for (const apiProviderName of Object.keys(state.apiProvidersByName)) {
+    const apiProvider = state.apiProvidersByName[apiProviderName];
+
+    // Ensure that the API provider address has enough ETH to create the onchain provider
+    await state.deployer.sendTransaction({ to: apiProvider.address, value: ethers.utils.parseEther('5') });
```

Comment:
There is also a `fundProviderAccounts()` function

Reply:
This is a leftover from before I refactored the functions and isn't needed so I'll remove. Thanks